e2e/tests/ui/pages/package-list/filter.spec.ts

```diff
@@ -18,12 +19,20 @@ test.describe("Filter validations", { tag: "@tier1" }, () => {
     // Full search
     await toolbar.applyTextFilter("Filter text", "keycloak-core");
     await table.waitUntilDataIsLoaded();
-    await table.verifyColumnContainsText("Name", "keycloak-core");
+    let tableRow = table.getRowsByCellValue({
+      Name: "keycloak-core",
+      Version: "18.0.6.redhat-00001",
+    });
+    await expect(await tableRow.count()).toBeGreaterThan(0);
 
     // Type filter
     await toolbar.applyMultiSelectFilter("Type", ["Maven", "RPM"]);
     await table.waitUntilDataIsLoaded();
-    await table.verifyColumnContainsText("Name", "keycloak-core");
+    tableRow = table.getRowsByCellValue({
+      Name: "keycloak-core",
+      Version: "18.0.6.redhat-00001",
+    });
+    await expect(await tableRow.count()).toBeGreaterThan(0);
```

Comment:
Same as above, let;s remove the first `await`

```suggestion
    expect(await tableRow.count()).toBeGreaterThan(0);
```

Reply:
Current code:
```ts
await expect(await tableRow.count()).toBeGreaterThan(0);
```

- The current code has 2 `await`. The first `await` has not effect. The second one does. 
- As you pointed out `count()` returns a promise, therefore the second `await` does apply correctly. But the first one does not.

Reply:
If we leave the current code it won't break anything. I suggested to remove the first `await` just in the spirit of having clean code and not get confused if in the future we review the code again... as mentioned before I think any standard IDE should be able to warn us about this kind of things.

<img width="565" height="289" alt="image" src="https://github.com/user-attachments/assets/c173df3e-6afd-4d5c-9164-29ab95fd05b0" />